x-pack/test/cases_api_integration/security_and_spaces/tests/common/configure/patch_configure.ts

```diff
@@ -190,6 +190,41 @@ export default ({ getService }: FtrProviderContext): void => {
       });
 
       it('should not patch a configuration with duplicated custom field keys', async () => {
+        const configuration = await createConfiguration(
+          supertest,
+          getConfigurationRequest({
+            overrides: {
+              customFields: [
+                {
+                  key: 'wrong_type_key',
+                  label: 'text',
+                  type: CustomFieldTypes.TEXT,
+                  required: false,
+                },
+              ],
+            },
+          })
+        );
+
+        await updateConfiguration(
+          supertest,
+          configuration.id,
+          {
+            version: configuration.version,
+            customFields: [
+              {
+                key: 'wrong_type_key',
+                label: '#1',
+                type: CustomFieldTypes.TOGGLE,
+                required: false,
+              },
+            ],
+          },
+          400
+        );
+      });
+
+      it('should not allow patching the type of a custom field', async () => {
```

Comment:
I think the description of this test is for the above test and vice-versa, correct?

Reply:
yup, you're right 😅